metricbeat/module/windows/perfmon/_meta/docs.asciidoc

```diff
@@ -15,12 +15,8 @@ to collect. The example below collects processor time and disk writes.
   metricsets: ["perfmon"]
   period: 10s
   perfmon.counters:
-    - group: "processor"
-      collectors:
-        - alias: "processor_time"
-          query: '\Processor Information(_Total)\% Processor Time'
-    - group: "disk"
-      collectors:
-        - alias: "bytes_written"
-          query: '\PhysicalDisk(_Total)\Disk Writes/sec'
+    - alias: "processor.processor_time"
```

Comment:
Probably this name should be changed to not stutter. Something like `processor.time.total.pct`?